apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -224,51 +232,51 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
 
         <Form {...form}>
           <form onBlur={onBlur}>
-            <SidebarContent>
-              <FormField
-                control={form.control}
-                name={'name'}
-                render={({ field }) => (
-                  <FormItem>
-                    <FormLabel>Name</FormLabel>
-                    <FormControl>
+            <SaveFormContext.Provider value={value}>
+              <SidebarContent>
+                <FormField
+                  control={form.control}
+                  name={'name'}
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Name</FormLabel>
                       <InputField>
-                        <Input placeholder="Untitled" {...field} disabled={isReadOnly} />
+                        <FormControl>
+                          <Input
```

Comment:
the only change here is that the inputs should be wrapped in the `FormControl` in order to have clicking on labels working